Add Github Issue Template

kieraneglin opened the pull request:
Does what it says on the tin! If I missed anything, please let me know.  I tried to follow the [issue submission guidelines](https://github.com/diesel-rs/diesel/blob/f7bb506ffc62ec72aafe60d7909d3614e3ff5376/CONTRIBUTING.md#submitting-bug-reports) as closely as possible.

For #969

killercup commented on .github/ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,39 @@
+Before submitting an issue, please check over the [issue tracker](https://github.com/diesel-rs/diesel/issues) to see if your problem has been discussed before.
```
This is good, but I fear the wall of text my scare people who do not want to report a bug. Maybe we can add something like this at the beginning?

> If you want to report a bug, we added some points below you can fill out. If you want to request a feature, feel free to remove all the irrelevant text. In addition to this issue tracker, you can also talk to Diesel maintainers and users on [Gitter](https://gitter.im/diesel-rs/diesel).

kieraneglin commented:
I've updated the template per your suggestions!  I also remove the `*Description*` lines.  They didn't provide anything meaningful, so I added spaces instead.

Eijebong commented on .github/ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,52 @@
+- **Rust:**
+- **Diesel:**
+- **Database:**
+- **Feature Crates:**
```
What does that mean ?

kieraneglin replied:
If you're using Chrono as a diesel feature, what version are you using?

I thought it'd be good to add the versions of any diesel "plug-ins" they may be using.

Eijebong replied:
Well if you're using the chrono feature in diesel, the chrono version it's using is using is set by diesel.

kieraneglin replied:
Ah!  Good point!  My internal Rust novice is showing.  Will fix

killercup commented:
Good changes. Let's merge this, then!

PeterW-LWL commented on .github/ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,51 @@
+### Steps to reproduce
+
+<!--
+Please include as much as your codebase as needed to reproduce the error.  If the relevant files are large, please consider linking to a public repository or a [Gist](https://gist.github.com/).
```
Please include as much **of** your codebase as needed...

killercup replied:
Good catch! Fixed in 1ad6ad8
